dsn.go

```diff
@@ -23,10 +24,13 @@ const (
 	defaultLoginTimeout           = 60 * time.Second  // Timeout for retry for login EXCLUDING clientTimeout
 	defaultRequestTimeout         = 0 * time.Second   // Timeout for retry for request EXCLUDING clientTimeout
 	defaultJWTTimeout             = 60 * time.Second
+	defaultJWTMaxRetries          = 10
 	defaultExternalBrowserTimeout = 120 * time.Second // Timeout for external browser login
 	defaultDomain                 = ".snowflakecomputing.com"
 )
 
+var defaultJWTRetryForCodes = [1]int{390144}
```

Comment:
Could we perhaps use `...` instead of the array length when initializing the array, e.g. `[...]int{390144}`, as it will infer the length of provided values.

Reply:
Yes, the value can also be omitted!